aether-producer/aether/producer/kernel_db.py

```diff
@@ -87,10 +120,18 @@ def __init__(self):
     def mode(self):
         return 'db'
 
-    def get_schemas(self):
+    def get_realms(self):
+        query = sql.SQL(_REALMS_SQL)
+        cursor = self._exec_sql('get_realms', 1, query)
+        return [row['realm'] for row in cursor]
+
+    def get_schemas(self, realm=None):
         self.last_check = datetime.now().isoformat()
         name = 'schemas_query'
-        query = sql.SQL(_SCHEMAS_SQL)
+        if realm:
+            query = sql.SQL(_SCHEMAS_SQL_SINGLE_REALM.format(realm=sql.Literal(realm)))
```

Comment:
```suggestion
            query = sql.SQL(_SCHEMAS_SQL_SINGLE_REALM).format(realm=sql.Literal(realm))
```

Reply:
I didn't see your comment in time and found that one the hard way :sob: